match/lib/match/nuke.rb

```diff
@@ -172,15 +172,15 @@ def delete_files!
     def certificate_type(type)
       cert_type = Spaceship.certificate.production
       cert_type = Spaceship.certificate.development if type == :development
-      cert_type = Spaceship.certificate.in_house if Match.enterprise? && Spaceship.client.in_house?
+      cert_type = Spaceship.certificate.in_house if type == :enterprise
```

Comment:
i am a bit confused is enterprise and in_house the same - if so, should all in_house things getting refactored and unified to enterprise ? 

and keep the `certificate.in_house` too, inside spaceship - for bc reasons?